pdns/pdns_recursor.cc

```diff
@@ -4784,7 +4786,9 @@ try
   while (!RecursorControlChannel::stop) {
     while(MT->schedule(&g_now)); // MTasker letting the mthreads do their thing
 
-    if(!(counter%500)) {
+    // Use primes, it avoid not being echeduled in casdes where the counter has a reguykar pattern.
```

Comment:
```suggestion
    // Use primes, it avoid not being scheduled in cases where the counter has a regular pattern.
```

Reply:
Thanks, that's what I get by editing & committing quickly before a call...